packages/compass-sidebar/src/components/sidebar.tsx

```diff
@@ -10,8 +10,8 @@ import { toggleIsGenuineMongoDBVisible } from '../modules/is-genuine-mongodb-vis
 import { changeFilterRegex } from '../modules/databases';
 import { updateAndSaveConnectionInfo } from '../modules/connection-info';
 
-// TODO: this should either change for this sidebar or be shared somewhere
-const initialSidebarWidth = spacing[4] * 10 + spacing[2]; // 248px
+// TODO: this should probably be shared be shared somewhere
+const initialSidebarWidth = spacing[6] * 4 - spacing[1]; // 252px
```

Comment:
Should we just align it right now and have this number in `ResizableSidebar` component? Is it that different between connection screen and this one at the moment?

Reply:
I'm still not sure if I'm allowed or expected to change the connection sidebar in this scope :)

Reply:
I think having one variable in `resizable-sidebar.tsx` like `defaultInitialSidebarWidth` and referencing that from both sidebars would be a nice simple way to do it. I think it's fine to update other things when we do things like this.

Reply:
OK I added defaults to resizable sidebar and removed the others.